service/framework.go

```diff
@@ -252,6 +253,18 @@ func NewVersionedResources(config Config, k8sClient kubernetes.Interface, awsCon
 
 		ccServiceConfig.Logger = config.Logger
 
+		// setup OIDC
```

Comment:
This looks super ugly, but if it will work with empty flags - I'll update cloudconfig then

Reply:
Do we want to add empty flags? Why can't the slice be empty if OIDC is not enabled?